api/api.yml

```diff
@@ -85,6 +85,110 @@ paths:
       responses:
         '200':
           description: computation stopped
+  /launcher/app-user/applications:
+    get:
+      tags:
+      - launcher
+      summary: Get applications in cockpit
+      responses:
+        '200':
+          description: Applications list
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ComputationApplication'
+    post:
```

Comment:
W Computation Cockpit nie można dodać nowej aplikacji, to jest z funkcjonalność z Library.

Reply:
Okej, w takim razie trochę źle to zrozumiałem. Myślałem, że możesz wybrać aplikacje jako "ulubione"

Reply:
I tak też traktuję te endpointy @PatrykCholewa

Reply:
:expressionless: